<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
    </property>
    <property name="title">
     <string>Select Items you enjoyed</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Items</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inputLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="addButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="inputList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>341</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="recButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Get Recommendations</string>
     </property>
    </widget>
    <widget class="QListWidget" name="algList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
     <item>
      <property name="text">
       <string>algorithm1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>algorithm2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>algorithm3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Algorithm</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="itemClearButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>381</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
    </property>
    <property name="title">
     <string>Recommendations:</string>
    </property>
    <widget class="QTextEdit" name="itemRecList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>341</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="clearAllButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Clear All</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>381</width>
      <height>311</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
    </property>
    <property name="title">
     <string>Select Items you enjoyed</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select users</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="userInputLine">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>30</y>
       <width>191</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="userAddButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>30</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="userInputList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>341</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
    <widget class="QPushButton" name="userRecButton">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>250</y>
       <width>141</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Get Similar Users</string>
     </property>
    </widget>
    <widget class="QListWidget" name="userAlgList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>141</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
     <item>
      <property name="text">
       <string>algorithm1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>algorithm2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>algorithm3</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Algorithm</string>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel_2">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>210</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="userClearButton">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>210</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Clear</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>330</y>
      <width>381</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(16, 16, 17);
background-color: rgb(199, 198, 201);</string>
    </property>
    <property name="title">
     <string>Similar Users</string>
    </property>
    <widget class="QTextEdit" name="userRecList">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>341</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 12pt &quot;Lucida Grande&quot;;
selection-color: rgb(0, 0, 0);</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="userClearAllButton">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>230</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Clear All</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>940</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
